pantheon-bundle/frontend/src/app/app.tsx

```diff
@@ -27,12 +27,16 @@ class App extends Component<any, IAppState> {
 
     const realFetch = self.fetch
     self.fetch = (input: RequestInfo, init?: RequestInit | undefined) => {
-      let newInput = input.toString()
-      if (window.location.host.startsWith('localhost') && input.toString().startsWith('/')) {
+      let newInput = input.toString();
+      if (
+        window.location.host.startsWith('localhost')
+        // Only changing requests Sling's old port
+        && parseInt(window.location.port) === 8080
+        && input.toString().startsWith('/')) {
         newInput = 'http://localhost:8181' + input.toString()
       }
       console.log('Development fetch', input, '=>', newInput)
-      return realFetch(newInput, init)
+      return realFetch(newInput, init);
```

Comment:
:exploding_head:

Reply:
All this JS without colons makes me itchy :joy: